seismicpro/utils/interactive_plot_utils.py

```diff
@@ -141,11 +135,31 @@ def __init__(self, *, plot_fn=None, click_fn=None, slice_fn=None, unclick_fn=Non
         self.fig.canvas.mpl_connect("button_release_event", self.on_release)
         self.fig.canvas.mpl_connect("key_press_event", self.on_press)
 
-        # Build plot box
-        self.title_widget = widgets.HTML(value="", layout=widgets.Layout(**TEXT_LAYOUT))
-        self.view_button = widgets.Button(icon="exchange", tooltip="Switch to the next view",
+        # Define widgets and toolbar buttons
+        # Non-toggle buttons have a one-space description to be aligned with toggle buttons, which append it even if
+        # only icon is defined, keep it as is until https://github.com/jupyter-widgets/ipywidgets/issues/2209 is fixed
```

Comment:
This comment is a bit unclear

Did I understand it correctly?

```suggestion
        # Adding ones-space description to non-toggle buttons in order to align them with toggle buttons, 
        # because toggle buttons always have a description even if only icon is defined in code.
        # This should be fixed after the corresponding ipywidgets issue is fixed: 
        # https://github.com/jupyter-widgets/ipywidgets/issues/2209
```